x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/monitoring/exporter/MonitoringTemplateUtils.java

```diff
@@ -29,7 +29,7 @@
      * <p>
      * It may be possible for this to diverge between templates and pipelines, but for now they're the same.
      */
-    public static final int LAST_UPDATED_VERSION = Version.V_7_0_0.id;
+    public static final int LAST_UPDATED_VERSION = Version.V_7_0_1.id;
```

Comment:
Shouldn't this be `7_1_0`?

Reply:
There's nothing wrong with making it that, but the last update relative to the 7.x line of code will be in 7.0.1 (once it's backported there). No change really happens in 7.0.

If someone were to upgrade from 7.0.0 to 7.1.x, it will have the same effect as upgrading from 7.0.0 to 7.0.x -- the templates will be replaced as expected. But if they go from 7.0.x (other than 7.0.0), then there's no need to replace the templates.

Reply:
Gotcha, makes sense!